app/cafe.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                "Visitor is not vaccinated."
+            )
+
+        expiration_date = visitor["vaccine"].get("expiration_date")

```

Comment:
do you need an external variable here?

Reply:
if  visitor["vaccine"].get("expiration_date") < date.today():